simulation_parameters/Constants.cs

```diff
@@ -435,6 +435,18 @@ public static class Constants
     /// </summary>
     public const float MUCILAGE_COOLDOWN_TIMER = 1.5f;
 
+    /// <summary>
+    ///   How many times cell gets slowed down with mucocyst (slime jet upgrade) on
+    /// </summary>
+    public const float MUCOCYST_SPEED_DIVISOR = 60;
```

Comment:
Usually speed modifiers are multipliers instead of divisors as dividing is a much more expensive floating point operation than multiplying.